src/registrar/models/portfolio.py

```diff
@@ -131,17 +131,25 @@ def portfolio_type(self):
         Returns a combination of organization_type / federal_type, seperated by ' - '.
         If no federal_type is found, we just return the org type.
         """
```

Comment:
Refactored the code here for get_federal_type and get_portfolio_type so that the same base logic will be called from the API as is already called in other contexts.